capstone/src/main/webapp/restaurantSingUp.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>STEP Capstone</title>
+    <script src="signUp.js"></script>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+  </head>
+  <body>
+    <h1>Sign Up as a Restaurant!</h1>
+    <div id="sign-up">
+      <p>Name:</p>
+      <textarea id="name"></textarea>
```

Comment:
Should all of these fields be `textarea`? It seems appropriate to use `textarea` in the case of the story, but that's about it. Would an `input` tag work better in most cases?

Reply:
Yup that's a good call. I left it for the cuisines as well just because that might be a bit longer as well.